backend/api/management/commands/seed.py

```diff
@@ -1,7 +1,11 @@
-from django.core.management.base import BaseCommand
+import json
 from argparse import ArgumentParser
+from pathlib import PosixPath
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
 
-from backend.api.client import client
+from backend.api.client import CollectionName, client
 
 # USAGE :
 #   python manage.py seed --mode=(refresh | clear)
```

Comment:
Maybe you should mention that `clear` is the default mode.

Reply:
Yep. But honestly, `clear` is not useful right now haha. I should remove it...

Reply:
it should just be `python3 manage.py seed`

Reply:
Hum, okay. I'm not against that